cmd/clusterctl/client/rollout_test.go

```diff
@@ -32,7 +32,7 @@ func Test_clusterctlClient_RolloutRestart(t *testing.T) {
 		client *fakeClient
 	}
 	type args struct {
-		options RolloutRestartOptions
+		options RolloutOptions
```

Comment:
Let's refactor this test so that we can assert on each of the rollout behaviors - pause, resume, restart.

Reply:
Done. PTAL